codalab/lib/worksheet_util.py

```diff
@@ -379,100 +375,6 @@ def friendly_render_dep(dep):
     return None
 
 
-def interpret_file_genpaths(client, requests):
```

Comment:
Good idea, I'll document it. The distinction is that worksheet_util doesn't make any calls to the model, they're kind of just like static helper functions.